modules/etcd-defrag.adoc

```diff
@@ -29,6 +29,11 @@ Verify that the defragmentation process is successful by viewing one of these lo
 * cluster-etcd-operator pod
 * operator status error log
 
+[WARNING]
+====
+Automatic defragmentation can cause leader election failures in various OpenShift core components, such as the kube-controller-manager, which triggers a restart of that component. The restart is harmless and either triggers failover to the next running instance, or the component resumes work again after the restart.
```

Comment:
s/failures/failure ?? 

Example"
_leader election failure... triggers a _

Not sure you need the comma after _instance_.